docs/serving/samples/cloudevents/cloudevents-vertx/pom.xml

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>org.knative.examples</groupId>
+    <artifactId>cloudevents-example-vertx</artifactId>
+    <version>1.0-SNAPSHOT</version>
+
+    <properties>
+        <java.version>1.8</java.version>
+    </properties>
+
+    <dependencies>
+        <dependency>
+            <groupId>io.cloudevents</groupId>
+            <artifactId>cloudevents-http-vertx</artifactId>
+            <version>2.0.0-SNAPSHOT</version>
```

Comment:
FYI since CE Java is in active development, using a SNAPSHOT here will most probably make Knative's CI unstable

Reply:
I don't think that this is actually _building_ all the samples.

Reply:
even worse then - the sample will get outdated very quickly.

Reply:
yeah, that is true! 

But not sure what consortium, would maintain all the different platforms on the docs repo :man_shrugging: 

Perhaps worth to bring it up at the TOC meeting? But I think that would over complicate the CI for the docs.

But thinking about it. Perhaps this makes sense in a more "samples focused" org? `knative-samples` ?

Reply:
I'd wait at least for the first milestone/rc release of the SDK, so that the version is locked. This way, it should remain buildable/runnable no matter how the SDK evolves in the future

Reply:
No strong opinion here, tbh

Reply:
Now it should be solved since we're using the milestone1